client/firewall/nftables/manager_linux.go

```diff
@@ -650,8 +677,8 @@ func (m *Manager) Flush() error {
 	// set must be removed after flush rule changes
 	// otherwise we will get error
 	for _, s := range m.setRemoved {
-		m.conn.FlushSet(s)
-		m.conn.DelSet(s)
+		m.rConn.FlushSet(s)
```

Comment:
is rConn intended to be used here?

Reply:
FlushSet - removes all elements from the set, and right after this operation, we delete it. So yes, we use rConn, sConn only to add/remove individual IPs.